### DISCUSSION TITLE
Add another value in the permissions table

### DISCUSSION THREAD

**Discussion by rafig256** [spatie/laravel-permission#2670] [General]
Considering that the number of my permissions is high, I need to categorize them. For this reason, I have manually added a column called group_name in the permissions table. Now I want to add a possibility to populate the group_name column when I use a command like php artisan permission:create-permission "permission_name".
How can I change the settings for permission:create-permission "permission_name"?

**Comment by parallels999** [comment]
You could make your custom command and override the existing one

https://github.com/spatie/laravel-permission/blob/84f3138432903636273e3a79cd56eff381fbd312/src/Commands/CreatePermission.php#L8-L24
